app/views/dashboard/index.html.erb

```diff
@@ -16,8 +16,32 @@
   </thead>
 
   <tbody>
-    <% @reasons.each do |reason| %>
-      <tr class="<%= "active text-muted inactive-reason" if reason.inactive %>">
+    <% @active_reasons.each do |reason| %>
+      <tr class="">
+        <td><%= link_to reason.reason_name, "/reason/#{reason.id}" %></td>
+        <td data-sort-value="<%= reason.post_count %>"><%= reason.post_count %></td>
+        <td><%= truncate((reason.last_post_title.nil? ? reason.posts.last.title : reason.last_post_title), :length => 40, :seperator => ' ') %></td>
```

Comment:
Use the new hash syntax where possible: instead of `{ :a => :b }`, use `{ a: :b }`.